.flake8

```diff
@@ -1,12 +1,14 @@
 [flake8]
-exclude=
-    .tox
-    setuptools/_vendor,
-    pkg_resources/_vendor
+max-line-length = 88
+exclude =
+	setuptools/_vendor
+	pkg_resources/_vendor
 ignore =
```

Comment:
It's preferable to use `extend-ignore` so that instead of overriding flake8's default ignores, you'd just append values to that list. This allows us to stop re-adding 503/504 in every project.

Reply:
https://github.com/PyCQA/pycodestyle/blob/master/pycodestyle.py#L84

Reply:
oh, nice. I was unaware of extend-ignore. That would be a great improvement to jaraco/skeleton. Filed as jaraco/skeleton#28.